Source/textdat.cpp

```diff
@@ -645,6 +645,38 @@ const TextDataStruct alltext[] = {
+	    1, 5, TSFX_BMAID17 },
+	{ "Beg pardon! But have you heard what has been happening? Some of the men vanished from the village. Deasus’ sons Eneas and Rauf claimed to see strange lights glowing deep within the woods a few days ago but no one has heard from them since. I too have seen a crimson  shimmering from beyond the ash groves but I’ve been too afraid to approach it. I fear that Ogden or our healer Pepin may be next. Won’t you please find out what has happened?"
+	    1, 5, TSFX_BMAID21 },
+	{ "Have you looked into what is causing the strange lights yet? There are more men missing since last we spoke and I’m beginning to fear that soon we may all vanish."
+	    1, 5, TSFX_BMAID22 },
+	{ "Demons that assume the form of beautiful women you say. That’s horrible! Thank the light that you solved this mystery before any more of our men were lore to their deaths. Thank you for keeping us safe from the power of darkness once again."
+	    1, 5, TSFX_BMAID23 },
+	{ "Pray your pardon, but I've something to tell you that you may find interesting. Was this strangest thing but drinking at the tavern, Farnham was rumbling about something called Azeroth he also said something about a fallen angel. It was hard to understand him because he was very drunk and disoriented but I seem to remember something about a key in a barrel he also kept covering his face and repeating the word Izual over and over again."
+	    1, 5, TSFX_BMAID24 },
+	{ "You know now that I think about it, maybe it wasn’t a key in a barrel but a barrel that was the key. Does that make any more sense?"
+	    1, 5, TSFX_BMAID25 },
+	{ "Azeroth and Izual... I don’t know why I didn’t remember that story earlier. I've heard that story often and often from my grandmother and Cain, the storyteller. Well, I hope that my information was still more of a help than a heritance. May light guide you my friend."
+	    1, 5, TSFX_BMAID26 },
+	{ "I know that my grandmother’s story seems strange but the graveyard does have many mysteries surrounding it. It couldn’t hurt you to put something there see what happens, could it? Maybe you will find out something that will help us all."
+	    1, 5, TSFX_BMAID28 },
+	{ "My grandmother had a dream about you last night, she said that in her dream you used your bare hands to defeat one of those fallen monsters that lurk under the church. When it died it exploded into a fabulous treasure! Although she was a bit frightened by her dream, she was able to describe the beast quite clearly to me."
+	    1, 5, TSFX_BMAID29 },
+	{ "My grandmother had that exact same dream again. She said you are quite brave and seemed very pleased with your prize."
+	    1, 5, TSFX_BMAID30 },
+	{ "I really don’t know much about priest Tremain, he never visits the tavern preferring to keep into the company of Pepin and Cain perhaps it’s because they two have more scholarly pursuits."

```

Comment:
`it’s because they two have more scholarly pursuits`

Based on this speech, I hear it as `they too have`. In reference that "they also" have scholarly pursuits, whereas for two you'd have to say `because those two have more scholarly pursuits`.

Reply:
For reference, you can see how the creator of Diablo evolution has dubbed these here: https://diablo-evolution.net/index.php?pageid=q1b10

```
I really don't know much about the priest Tremain. He never visits the tavern, preferring to keep to the company of Pepin and Cain. Perhaps it is because they, too, have more scholarly pursuits.
```

Reply:
@galaxyhaxz 
> `it’s because they two have more scholarly pursuits`
> 
> Based on this speech, I hear it as `they too have`. In reference that "they also" have scholarly pursuits, whereas for two you'd have to say `because those two have more scholarly pursuits`.

Oh, makes much more sense, I will update it!

Thank you for the link! Who could tell that there is much more story to be told by Diablo 1?

Reply:
Yes, there are many cut quests (wirts treasure, infested cellar, map of doom, shadowfang, grimspike, andariel, horizon). It's a shame they weren't given another year to work on the game or it's probably be much different.

And maybe the code would look like it was written by adults xD